spec/models/lesson_spec.rb

```diff
@@ -66,27 +69,35 @@
     it "lesson belongs to @course" do
       expect(@lesson.course).to eq(@course)
     end
+
+    it "lesson_with_document" do
+      expect(Lesson.last.documents).to eq([@document])
```

Comment:
Similar to previous test.  Try something like `@lesson.documents.include? @document`  It will assure that the lesson we created has that document we created and Lesson is the class.  So it's kind of testing the last document created belongs to a lesson instead of @lesson.